tap_closeio/streams.py

```diff
@@ -165,13 +165,26 @@ def sync_activities(ctx):
     # calls that are in progress _while_ an extraction is happening, no
     # matter the replication frequency or call duration.
     offset_secs = ctx.config.get("activities_window_seconds", (60 * 60 * 24))
+    date_window = int(ctx.config.get("activities_date_window", 15))
```

Comment:
```suggestion
    date_window = int(ctx.config.get("date_window", 15))
```

Reply:
Also add default value comment above this line.

Reply:
Added.